src/js/components/shepherd-modal.svelte

```diff
@@ -120,15 +127,23 @@
    * @private
    */
   function _styleForStep(step) {
-    const { modalOverlayOpeningPadding, modalOverlayOpeningRadius } = step.options;
+    const {
+      modalOverlayOpeningPadding,
+      modalOverlayOpeningRadius
+    } = step.options;
 
-    if (step.target) {
+    if (step) {
```

Comment:
It looks like this is the main change? We call positionModal now regardless of whether there is a target? I think the `closeModalOpening` stuff maybe needs another look.